lib/completions/endpoints/hugging_face.rb

```diff
@@ -19,9 +19,7 @@ def provider_id
         private
 
         def model_uri
-          URI(SiteSetting.ai_hugging_face_api_url).tap do |uri|
-            uri.path = @streaming_mode ? "/generate_stream" : "/generate"
-          end
+          URI(SiteSetting.ai_hugging_face_api_url)
```

Comment:
Moving to headers instead of path makes it compatible with both self-hosting and hosted API